mw-webapp/src/logic/reportsTable/columns.tsx

```diff
@@ -15,13 +15,7 @@ const DEFAULT_SUMMARY_TIME = 0;
 const columnHelper = createColumnHelper<DayReport>();
 
 /**
- * Determines which columns will be in the table, the values in the cells and what types of data can be rendered in cells
- * The tanstack table has a bug about typing columns:
- * https://github.com/TanStack/table/issues/4382
- * According to creators should only be using the column helper and not pre-typing columns
- * We can add type as:
- * ColumnDef<DayReport, Date & JobDone[] & PlanForNextPeriod[] & CurrentProblem[] & MentorComment[] & string[] & boolean>
- * but it's not recommend by creators
+ * Table columns
  */
 export const columns = [
```

Comment:
Why you deleted comments? If you found how resolve this poblem then add column typing please